lib/session.go

```diff
@@ -252,6 +276,10 @@ func (s *Session) Variables() *Variables {
 				"AWS_SECURITY_TOKEN",
 			)
 		}
+
+		if s.AWSCreds.Region != nil && *s.AWSCreds.Region != "" {
+			vars.Set["AWS_DEFAULT_REGION"] = *s.AWSCreds.Region
```

Comment:
Same question here about `AWS_REGION` vs `AWS_DEFAULT_REGION`

Reply:
As mentioned in my other comment, AWS tools don't appear to have standardized on one variable or the other. I'll update this to export both.